contribute/backend/upgrading-go-version.md

```diff
@@ -0,0 +1,17 @@
+# Upgrading Go Version
+
+Notes on upgrading Go version.
+
+Example PR: https://github.com/grafana/grafana/pull/79329
+
+## The main areas that need to change during the upgrade are:
+- https://github.com/grafana/grafana/blob/d8ecea4ed93efb2e4d64a5ee24bc08f3805f413d/scripts/drone/variables.star#L6
+- https://github.com/grafana/grafana/blob/d8ecea4ed93efb2e4d64a5ee24bc08f3805f413d/Makefile#L264
+- https://github.com/grafana/grafana/blob/d8ecea4ed93efb2e4d64a5ee24bc08f3805f413d/Dockerfile#L6
+
+### Additional files to change
+- Take a look in `.github/workflows` folder for what `go` version is being used there in various workflows.
```

Comment:
I wonder if there is a way to define a variable once and refer to this from the workflows.

Reply:
I only added the workflows as courtesy to keep an eye on, they are not necessarily have to have the same version as the main grafana. At least not all of them. 

Sharing a variable or a config is what I would rather avoid, to let them be separate and not too coupled to each other.